src/main/java/com/eafit/herzon/teis/models/Cart.java

```diff
@@ -0,0 +1,132 @@
+package com.eafit.herzon.teis.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+import org.hibernate.annotations.UpdateTimestamp;
+
+
+/**
+* comment.
+*/
+@Entity
+@Table(name = "carts")
+public class Cart {
+  /**
+  * The ID of the cart.
+  */
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  /**
+  * Comment.
+  */
+  @OneToMany(fetch = FetchType.LAZY, mappedBy = "cart",
```

Comment:
FetchType.EAGER. I reccomned using EAGER in case you are constantly retrieving CartItems for Cart
```suggestion
  @OneToMany(fetch = FetchType.EAGER, mappedBy = "cart",
```